### ISSUE TITLE
RFM - SikuliX 1.1.0

### ISSUE THREAD

**Issue by RaiMan** [chocolatey/chocolatey-package-requests#456]
https://chocolatey.org/packages/sikulix

As the owner of the underlying SikuliX (https://github.com/RaiMan/SikuliX1) I do not want to maintain this package (not familiar with Chocolatey and do not want to be), but I am willing to support new maintainer(s)

**Comment by dgalbraith** [comment]
Working with @RaiMan on this - I have been assigned as a package maintainer and will push a fresh package once available.

**Comment by RaiMan** [comment]
@dgalbraith 
Is it possible, to already now with available version 2.0.4 create a new package?
2.0.5 will still take a while.

**Comment by dgalbraith** [comment]
Yes - everything needed is in place and I will create a package for the current 2.0.4 version.

**Comment by AdmiringWorm** [comment]
@dgalbraith any update on this?

**Comment by AdmiringWorm** [comment]
@dgalbraith Do you have any update on this request?
If there is no update within the next 7 days I will move this request back to being available to new maintainers.

**Comment by dgalbraith** [comment]
@AdmiringWorm in progress.  Core package is complete but have been trying to build out automated installs of some optional dependencies that I have hit a few blockers on.

**Comment by pauby** [comment]
@dgalbraith Any update to this?

**Comment by AdmiringWorm** [comment]
Moving request to `Available for Maintainer(s)` due to no response.
